core/modules/views_ui/views_ui.admin.inc

```diff
@@ -2978,7 +2978,7 @@ function views_ui_reorder_displays_form($form, &$form_state) {
       'weight' => array(
         '#type' => 'weight',
         '#value' => $display->position,
-        '#delta' => count($last_display->position),
+        '#delta' => (empty($last_display->position)) ? 1 : 0,
```

Comment:
This would result in a select list that only had either 0 or 3 options (-1, 0, and 1). I think the right thing here is probably:
```suggestion
        '#delta' => count($view->display),
```